qiskit_braket_provider/providers/adapter.py

```diff
@@ -128,6 +129,7 @@
     "zz": lambda angle: [braket_gates.ZZ(2 * pi * angle)],
     # Global phase
     _GPHASE_GATE_NAME: lambda phase: [braket_gates.GPhase(phase)],
+    "measure": lambda: [measure.Measure()],
```

Comment:
Is this necessary? This dict is only used for gates.
```suggestion
```

Reply:
This is used to match the qiskit `measure` to the Braket `Measure` so without it, the Braket measure isn't added to the Circuit. 

I do agree that this might not work well in this particular dict. Maybe there can be a `_ADDITIONAL_INSTRUCTION_NAME_TO_BRAKET_NAME` dict for things like measure?

Reply:
That should be handled here: https://github.com/qiskit-community/qiskit-braket-provider/blob/0751b4c33f8f4cf262e273064eb241eb5e959a9d/qiskit_braket_provider/providers/adapter.py#L434-L437

Reply:
Gotcha, that does work for the Qiskit to Braket conversion but the Braket to Qiskit conversion [here](https://github.com/qiskit-community/qiskit-braket-provider/blob/main/qiskit_braket_provider/providers/adapter.py#L571-L573) checks for gates in the _GATE_NAME_TO_QISKIT_GATE dict. 

Is there a better way to implement the Braket to Qiskit conversion for measure?

Reply:
Yeah, it's fine to leave it in the Braket->Qiskit direction; you could do a special check for measure operators, but that's not necessary.

Reply:
Removed the `measure` gate from `_GATE_NAME_TO_BRAKET_GATE` but left it in `_GATE_NAME_TO_QISKIT_GATE`